config/configmodels/configmodels.go

```diff
@@ -126,13 +126,24 @@ type Pipelines map[string]*Pipeline
 // These are helper structs which you can embed when implementing your specific
 // receiver/exporter/processor config storage.
 
+// BackPressureState defines if backpressure should be exterted or not.
+type BackPressureState bool
```

Comment:
I find the name `BackPressureState` a bit misleading. It sounds as if it is the _current_ state of backpressure, i.e. if the receiver is currently under a backpressure situation. Perhaps rename to `BackPressureSetting`?